pyanaconda/modules/common/constants/interfaces.py

```diff
@@ -58,3 +58,13 @@
 DNF_HANDLER = DBusInterfaceIdentifier(
     namespace=DNF_NAMESPACE
 )
+
+PAYLOAD_SOURCE = DBusInterfaceIdentifier(
+    namespace=PAYLOAD_SOURCE_NAMESPACE,
```

Comment:
Shouldn't we use PAYLOAD_NAMESPACE here?
I mean ... so that the interface is Modules.Payload.Source, not Modules.Payload.Source.Source ?

Reply:
Fixed.